main/acle.md

```diff
@@ -9276,12 +9279,14 @@ Scatter store quadwords.
    // _mf8, _bf16, _f16, _f32, _f64
    void svst1q_scatter[_u64base][_s8](svbool_t pg, svuint64_t zn, svint8_t data);
    void svst1q_scatter[_u64base]_offset[_s8](svbool_t pg, svuint64_t zn, int64_t offset, svint8_t data);
-   void svst1q_scatter_[u64]offset[_s8](svbool_t pg, const uint8_t *base, svuint64_t offset, svint8_t data);
+   void svst1q_scatter_[s64]offset[_s8](svbool_t pg, uint8_t *base, svint64_t offset, svint8_t data);
+   void svst1q_scatter_[u64]offset[_s8](svbool_t pg, uint8_t *base, svuint64_t offset, svint8_t data);
```

Comment:
```suggestion
   void svst1q_scatter_[s64]offset[_s8](svbool_t pg, int8_t *base, svint64_t offset, svint8_t data);
   void svst1q_scatter_[u64]offset[_s8](svbool_t pg, int8_t *base, svuint64_t offset, svint8_t data);
```
Minor detail but I see that the `svld1q_gather_` intrinsic is documented this way.